scripts/modules/elastic_stack.py

```diff
@@ -702,6 +705,12 @@ def add_arguments(cls, parser):
             help="kibana elasticsearch output url(s)."
         )
 
+        parser.add_argument(
+            "--kibana-apm-disable-servicemaps",
```

Comment:
all the "don't do something" flags read like `--no-do-something`, what about following suit here?

Reply:
(asking because i always fail to remember `--skip-downlad` as a matter of fact :grimacing:

Reply:
great call, updated